google/__init__.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Is it safe to remove ?

Reply:
Yes, the import fails with protobuf `4.21.1` if we don't remove it. I can publish a pre-release version with this change. WDYT?